index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="./cssstyles/home.css">
+    <link href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined" rel="stylesheet" />
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&family=Roboto:wght@400;500;700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+
+    <script src="./javascript/home.js" defer></script>
+
+    <title>BUSINESS SEMINAR</title>
+
+</head>
+
+<body>
+    <header class="header">
+        <nav class="nav-container">
+            <div class="top-nav">
+                <a href="#"><img src="./assets/icons/facebook.png" alt="facebook"></a>
+                <a href="#"><img src="./assets/icons/twitter.png" alt="twitter"></a>
+                <span class="span">English</span>
+                <span class="span">My Page</span>
+                <span class="span">Logout</span>
+            </div>
+            <div class="bottom-nav">
+
+                <div class="hamburger-menu">
+                    <span></span>
+                    <span></span>
+                    <span></span>
+                </div>
+
+                <div class="nav-logo-container">
+                    <img class="nav-logo" src="./assets/logos/headerlg.png" alt="seminar logo">
```

Comment:
- [ ] [OPTIONAL]  I think I used different logos in index and about pages. It would be cool you used just one. See difference form these images below

Index
![image](https://user-images.githubusercontent.com/69011963/176916691-2e668ebd-200c-43b7-97bc-fe400a85e50c.png)

About
![image](https://user-images.githubusercontent.com/69011963/176916795-85f43756-6a7b-4ffd-bf88-63e0c48deafb.png)